change/@internal-react-composites-2b6765de-ed42-4658-80bf-fa34b781fb89.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
```

Comment:
This PR is not really changing the API. This changelog should be none type.
I believe that is due to a bug, we forgot to change the exported function signature to include alternative Caller id?
Could you link to the PR that fixes this issue?